src/services/expenses.service.js

```diff
@@ -0,0 +1,67 @@
+const { getNumberId } = require('./../utils');
+
+class Expenses {
+  constructor(initialExpenses) {
+    this._expenses = initialExpenses || [];
+  }
+
+  getAll({ userId, categories, from, to }) {
+    let res = this._expenses;
+
+    if (userId) {
+      res = res.filter((expense) => expense.userId === userId);
+    }
+
+    if (categories) {
+      res = res.filter((expense) => {
+        return categories === expense.category;
```

Comment:
The `getAll` method currently checks if `categories` is equal to `expense.category`. If `categories` is intended to be an array of categories, you should use `categories.includes(expense.category)` to filter expenses by multiple categories.